test/positive_approx_tests.jl

```diff
@@ -90,10 +90,11 @@ end
     order_unit_1 = reshape([one(RC₃)], 1, 1)
     m = LowCohomologySOS.sos_problem_matrix(M_1, order_unit_1)
     output_1 = sprint(print, m)
+    equality_sign = Sys.iswindows() ? "==" : "="
     @test occursin("Subject to", output_1)
-    @test occursin("-P[1,1] - 2 P[2,3] - λ = 0.0", output_1)
-    @test occursin("-2 P[1,2] - P[3,3] = -1.0", output_1)
-    @test occursin("-P[2,2] - 2 P[1,3] = 0.0", output_1)
+    @test occursin("-P[1,1] - 2 P[2,3] - λ "*equality_sign*" 0.0", output_1)
+    @test occursin("-2 P[1,2] - P[3,3] "*equality_sign*" -1.0", output_1)
+    @test occursin("-P[2,2] - 2 P[1,3] "*equality_sign*" 0.0", output_1)
     @test occursin("sdp : [P[1,1]  P[1,2]  P[1,3];", output_1)
     @test occursin("P[1,2]  P[2,2]  P[2,3];", output_1)
     @test occursin("JuMP.PSDCone()", output_1)
```

Comment:
we probably just need to test that
* We're maximizing `λ`
* `P` is the name of matrix variable
* there is `P[3,3]` but not `P[4,4]` (i.e. the size of `P` is correct)
* `sdp` is the name of `PSDCone()` constraint

so that's just 4 tests